go/rpcclientlib/README.md

```diff
@@ -1 +1,16 @@
-This package contains library code to allow client applications to connect to Obscuro nodes via RPC.
\ No newline at end of file
+This package contains library code to allow client applications to connect to Obscuro nodes via RPC.
+
+### Viewing keys
+
+Viewing keys are generated inside the wallet extension (or other users of the obscuro rpc client), and then signed by the wallet (e.g. MetaMask)
+to which the keys relate.
+The keys are then are sent to the enclave via RPC and processed by:
+- checking the validity of the signature over the viewing key
+- finding the account to which this viewing key corresponds
+We can do that by retrieving the signing public key from the signature.
+By hashing the public key, we can then determine the address of the account.
+- finally the enclave will save the viewing key (which is a public key) against the account, and use it to encrypt any
+sensitive requests (e.g. "eth_call" and "eth_getBalance") permitted to be viewed by that account
```

Comment:
Think you need to indent this a bit as well, or it won't be considered part of the bullet